### ISSUE TITLE
Angular5 upgrade: WaveSurfer undefined

### ISSUE THREAD

**Issue by davidlirving** [Alveo/alveo-transcriber#6]
```
ERROR TypeError: Cannot read property 'create' of undefined
    at PlayerComponent.ngOnInit (player.component.ts:110)
    at checkAndUpdateDirectiveInline (core.js:12400)
    at checkAndUpdateNodeInline (core.js:13927)
    at checkAndUpdateNode (core.js:13870)
    at debugCheckAndUpdateNode (core.js:14763)
    at debugCheckDirectivesFn (core.js:14704)
    at Object.eval [as updateDirectives] (AnnotatorComponent.html:10)
    at Object.debugUpdateDirectives [as updateDirectives] (core.js:14689)
    at checkAndUpdateView (core.js:13836)
    at callViewAction (core.js:14187)
```
All browsers/platforms

**Comment by davidlirving** [comment]
Consider moving to requestAnimationFrame() as per
https://github.com/katspaugh/wavesurfer.js/issues/1200

**Comment by davidlirving** [comment]
Fixed in 0e0688425244617dc9b1bdc6c16805971cdf886b

Angular5 doesn't support importing via the old Angular4 implementation.
